.github/workflows/pr_build.yaml

```diff
@@ -263,9 +309,13 @@ jobs:
           name: scratch-images
           path: .
       - name: Load archived images
-        run: zcat images.tar.gz | docker load
+        run: |
+          tar xvf images.tar.gz
+          make load-images
       - name: Load archived scratch images
-        run: zcat scratch-images.tar.gz | docker load
+        run: |
+          tar xvf scratch-images.tar.gz
+          make load-scratch-images
```

Comment:
Uses regctl to load the image for the architecture of this github runner into docker.